homework/pages/AnsForm.js

```diff
@@ -0,0 +1,12 @@
+class AnsForm {
```

Comment:
what was the reason to separate the same page into three small sections instead of describing everything in one class for that page?

Reply:
I thought that would be easier to understand the code.